C01/project-metadata/C01-ProjectId612-metadata.json

```diff
@@ -69,12 +69,44 @@
       }
     },
     "type": "Feature",
+    "place_name": "Taita–Taveta, Kenya",
     "geometry": {
       "type": "Point",
       "coordinates": [
         38.802647,
         -3.910022
       ]
     }
-  }
+  },
+  "regen:previewPhoto": {
```

Comment:
i think this is a field we need to add to the C01 project schema..

Reply:
I'm doing that as part of https://app.zenhub.com/workspaces/regen-registry-5f8998bec8958d000f4609e2/issues/regen-network/regen-registry/909